cmd/dogstatsd/subcommands/start/command.go

```diff
@@ -102,10 +107,35 @@ func RunDogstatsdFct(cliParams *CLIParams, defaultConfPath string, defaultLogFil
 		dogstatsd.Bundle,
 		forwarder.Bundle,
 		fx.Provide(defaultforwarder.NewParams),
+		// injecting the aggregator demultiplexer to FX until we migrate it to a proper component. This allows
+		// other already migrated components to request it.
+		fx.Provide(func(config config.Component, log log.Component, sharedForwarder defaultforwarder.Component) (*aggregator.AgentDemultiplexer, error) {
```

Comment:
Would it make sense to extract this into a named function, to keep the component and bundle list manageable?